content/troubleshooting/net/issues/AnotherIISProfiler.md

```diff
@@ -17,3 +17,5 @@ The CLR Profiling API only allows a single agent to be attached to a .NET proces
 
 To resolve the issue, Contrast implemented profiler chaining that allows Contrast to run side-by-side with another profiling tool. To enable profiling, see the `ProfilerChainingEnabled` configuration setting](installation-netconfig.html#overview).
 
+<!-- Should we update this with the equivalent common config property?  -->
+
```

Comment:
@DanFiedler See commented text above

Reply:
Yes we should replace that legacy configuration with agent.dotnet.enable_chaining